internal/osutils/osutils.go

```diff
@@ -32,22 +33,28 @@ func CmdString(c *exec.Cmd) string {
 // BashifyPath takes a windows style path and turns it into a bash style path
 // eg. C:\temp becomes /c/temp
 func BashifyPath(absolutePath string) (string, error) {
-	if absolutePath[0:1] == "/" {
+	if len(absolutePath) > 0 && absolutePath[0:1] == "/" {
 		// Already the format we want
 		return absolutePath, nil
 	}
 
-	if absolutePath[1:2] != ":" {
+	if len(absolutePath) < 2 || absolutePath[1:2] != ":" {
 		// Check for windows style paths
 		return "", errs.New("Unrecognized absolute path format: %s", absolutePath)
 	}
 
+	if strings.Contains(absolutePath, ";") {
+		return "", errs.New("Path is a list: %s", absolutePath)
+	}
```

Comment:
This will need to account for escaped characters if we feel we really need to assert this.

```
❯ touch "/tmp/foo;bar"
❯ ls /tmp/ | grep foo
foo;bar
```

Reply:
This is no longer needed.